src/Phinx/Console/Command/SeedRun.php

```diff
@@ -104,7 +105,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // run the seed(ers)
         $start = microtime(true);
-        $this->getManager()->seed($environment, $seed);
+        foreach($seedSet as $seed) {
+            $this->getManager()->seed($environment, trim($seed));
```

Comment:
That `trim()` causes a problem when a developer wants to run all seeds. Previously it was possible by not passing any argument to the command (`$seed` was `null` and it was passed like this to `seed()` method).

At the moment, `trim(null)` causes that an empty string is passed to that method causing:

```
[InvalidArgumentException]
The seed class "" does not exist
```